lib/base/number.js

```diff
@@ -17,50 +17,89 @@ comb.isNumber = function(obj) {
+ * Rounds a number to the specified places, rounding up.
+ *
+ * @example
+ * comb.number.roundCeil(10.000001, 2); //10.01
+ * comb.number.roundCeil(10.000002, 5); //10.00001
+ * comb.number.roundCeil(10.0003, 3); //10.001
+ * comb.number.roundCeil(10.0004, 2); //10.01
+ * comb.number.roundCeil(10.0005, 3); //10.001
+ * comb.number.roundCeil(10.0002, 2); //10.01
+ *
+ * @param {Number} num the number to round.
+ * @param {Number} precision the number of places to round to.
+ * @static
+ * @memberOf comb.number
  */
-comb.number = {
-    /**@lends comb.number*/
+function roundCeil(num, precision){
+    return Math.ceil(num * Math.pow(10, precision))/Math.pow(10, precision);
+}
 
 
-    /**
-     * Rounds a number to the specified places.
-     *
-     * @example
-     *
-     * comb.number.round(10.000009, 2); //10
-     * comb.number.round(10.000009, 5); //10.00001
-     * comb.number.round(10.0009, 3); //10.001
-     * comb.number.round(10.0009, 2); //10
-     * comb.number.round(10.0009, 3); //10.001
-     *
-     * @param {Number} num the number to round.
-     * @param {Number} places the number of places to round to.
-     */
-    round : function(number, places, increment) {
-        increment = increment || 1e-20;
-        var factor = 10 / (10 * (increment || 10));
-        return (Math.ceil(factor * +number) / factor).toFixed(places) * 1; // Number
-    },
+/**
+ * Rounds a number half down to the specified places.
+ * @example
+ *
+ * comb.number.roundHalfDown(0.225, 2);     //0.22
+ * comb.number.roundHalfDown(10.384, 2);    //10.38
+ * comb.number.roundHalfDown(10.386, 2);    //10.38
+ * comb.number.roundHalfDown(10.3869, 3);   //10.386
+ * comb.number.roundHalfDown(10.3861, 3);   //10.386
+ * comb.number.roundHalfDown(10.269019, 5); //10.26901
+ *
+ * @param {Number} num the number to round.
+ * @param {Number} precision the number of places to round to.
+ * @static
+ * @memberOf comb.number
+ */
+function roundHalfDown(num, precision) {
+    var multiplier = pow(10, precision);
+    return Math.floor(num * multiplier) / multiplier;
+}
 
 
-    /**
-     * Rounds a number to the specified places, rounding up.
-     *
-     * @example
-     *
-     * comb.number.roundCeil(10.000001, 2); //10.01
-     * comb.number.roundCeil(10.000002, 5); //10.00001
-     * comb.number.roundCeil(10.0003, 3); //10.001
-     * comb.number.roundCeil(10.0004, 2); //10.01
-     * comb.number.roundCeil(10.0005, 3); //10.001
-     * comb.number.roundCeil(10.0002, 2); //10.01
-     *
-     * @param {Number} num the number to round.
-     * @param {Number} places the number of places to round to.
-     */
-    roundCeil : function(number, places){
-         return Math.ceil(number * Math.pow(10, places))/Math.pow(10, places);
+/**
+ * Rounds a number half up to the specified places.
+ * @example
+ *
+ * comb.number.roundHalfUp(0.225, 2);     //0.23
+ * comb.number.roundHalfUp(10.384, 2);    //10.38
+ * comb.number.roundHalfUp(10.386, 2);    //10.39
```

Comment:
shouldnt this number round up to `10.37` since the 6 would cause the number to go up?